cypress/integration/navigationFlows/homePageLeftSidebar.spec.js

```diff
@@ -0,0 +1,31 @@
+describe('Home Page Left Sidebar', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should open the "More" links', () => {
+    // Go to home page
+    cy.visit('/');
+
+    // Click on the More... button in the nav
+    cy.get('#page-content-inner').within(() => {
+      cy.findByText('More...').click();
+      cy.findByText('More...').should('not.be.visible');
```

Comment:
refactor (non-blocking): chain Cypress commands since a reference to the element will already exist.

```suggestion
      cy.findByText('More...').click().should('not.be.visible');
```